tracer/src/Datadog.Trace/SpanContext.cs

```diff
@@ -145,6 +152,16 @@ private SpanContext(ulong? traceId, string serviceName)
         /// </summary>
         internal int? SamplingPriority { get; }
 
+        /// <summary>
+        /// Gets or sets the raw traceId (to support > 64bits)
+        /// </summary>
+        internal string RawTraceId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the raw spanId
+        /// </summary>
+        internal string RawSpanId { get; set; }
+
```

Comment:
These shouldn't ever change right? Should we make them readonly and set in the constructor?

Reply:
done in [905b2ab](https://github.com/DataDog/dd-trace-dotnet/pull/2536/commits/905b2ab40371a756240f91f452c608248ce60b71)